ocamltest/Makefile

```diff
@@ -33,8 +33,16 @@ else
 endif
 
 ifeq "$(filter $(UNIXLIB),$(OTHERLIBRARIES))" ""
+  ocamltest_unix := dummy
+	unix_name :=
+	unix_path :=
   unix := None
+	unix_include :=
 else
+  ocamltest_unix := real
+	unix_name := unix
+	unix_path := $(ROOTDIR)/otherlibs/$(UNIXLIB)
+	unix_include := -I $(unix_path) $(EMPTY)
   ifeq "$(UNIX_OR_WIN32)" "win32"
```

Comment:
Not sure if it is an artifact of Github's UI, but there seems to be a mixture of spaces and tabs in this diff.

Reply:
It's not an artefact - I've just fixed it!